.travis.yml

```diff
@@ -1,10 +1,14 @@
 language: python
-
+sudo: false
 python:
-- '2.7'
-
-install: 
+  - "2.6"
+  - "2.7"
+  - "3.2"
```

Comment:
sure you want to include py 26 and 3.2? 

These often lead to (imho) ugly code because of missing features